src/main/resources/application.yml

```diff
@@ -51,5 +51,6 @@ graylog:
   host: ${IEXEC_CORE_GRAYLOG_HOST:localhost}
   port: ${IEXEC_CORE_GRAYLOG_PORT:12201}
 
-debug:
-  forceTeeDisabled: ${IEXEC_DEBUG_FORCE_TEE_DISABLED:false} # this for dev mode only
+tee:
+  sgx:
+    mode: ${IEXEC_WORKER_SGX_MODE:OFF}
```

Comment:
How do we document this for end-users ?
Shouldn't we provide an explanation on the application configuration properties in README.md ?

Reply:
I suggest updating the semantic to `SGX_DRIVER_MODE` (or anything else) to avoid any confusion with official _SGX debug mode enclaves_.
In this way, `OFF` doesn't really mean something (the driver won't be off), we can replace it to `NONE` (open to any suggestion).
Recap idea:
`IEXEC_WORKER_SGX_DRIVER_MODE={NONE, LEGACY, NATIVE}

Reply:
@jbern0rd: You're absolutely right. Let's add a `Configuration` section in README.md.

@jeremyjams: Thanks for the proposal. I'll update my code with this suggestion, it seems indeed better.

Reply:
@jbern0rd: See https://github.com/iExecBlockchainComputing/iexec-worker/pull/432/commits/37a1556f046fe9a641183a160b2b9dc2559afbb9.
@jeremyjams: See https://github.com/iExecBlockchainComputing/iexec-worker/pull/432/commits/16d1e07cfba2d1507c961c0e19727cf82b8a380b.